README.md

```diff
@@ -49,6 +49,16 @@ For help and more options, type:
 page-computed-style -h
 ```
 
+## Installing & Using Locally
```

Comment:
I'd like to keep the readme as small and focused as possible. I expect most people will install the package globally, so I wouldn't add instructions for installing and using locally in the project readme. Instead we could create a separate document file or add an entry to the wiki. What do you think?

Reply:
👍

Reply:
A separate file will be easier to manage than a wiki entry I believe. I will update accordingly. Thanks